lib/typescript/render/providers/chatplugin/ChatPluginRender.tsx

```diff
@@ -96,6 +96,11 @@ function mapContent(message: Message): ContentUnion {
       text: messageContent.text,
     };
   }
+
+  return {
+    type: 'text',
+    text: 'Unknown message type',
+  };
 }
```

Comment:
added this return value when the message type is unknown in the chat_plugin render (otherwise we'll get an error for unknown message types)